Fix nat-service

aledsage opened the pull request:
The support in `NatService` for multi-action (i.e. passing in a delta to be applied all at once) is laying the ground work for the approach of having a micro-service that is passes the changes and applies those with appropriate locking (potentially from multiple Brooklyn instances).

andreaturli commented on vcloud-director/src/main/java/brooklyn/networking/vclouddirector/NatService.java:
```diff
@@ -123,6 +109,34 @@ public NatService build() {
     
+    public static class Delta {
+        private final List<PortForwardingConfig> toClose = Lists.newArrayList();
+        private final List<PortForwardingConfig> toOpen = Lists.newArrayList();
```
are `toClose` and `toOpen` really needed or we could just have one?

aledsage replied:
@andreaturli not sure I follow your suggestion. This says which ports mappings are to be created, and which are to be deleted.

andreaturli replied:
@aledsage agreed, in fact at least `updatePortForwardingImpl(Delta delta)` uses both. Ignore my comment.

nakomis commented on vcloud-director/src/main/java/brooklyn/networking/vclouddirector/PortForwarderVcloudDirector.java:
```diff
@@ -242,6 +243,21 @@ private NatService getService() {
+        // jclouds endpoint has suffix "/api"; but VMware SDK wants it without "api"
+        String convertedUri;
+        try {
+        	URI uri = URI.create(endpoint);
```
Double-indented (or tabs)

nakomis commented on vcloud-director/src/test/java/brooklyn/networking/vclouddirector/NatServiceLiveTest.java:
```diff
@@ -178,4 +178,22 @@ public Void call() throws Exception {
+        // jclouds endpoint has suffix "/api"; but VMware SDK wants it without "api"
+        String convertedUri;
+        try {
+        	URI uri = URI.create(endpoint);
```
Double-indented (or tabs)

aledsage commented:
Thanks @andreaturli @nakomis - I've incorporated your comments, and also replaced all tabs with spaces across all `*.java` files in repo (in a separate commit).

Merging now.
